index.html

```diff
@@ -227,10 +223,22 @@ <h3>Skills</h3>
     </div>
 </div>
     </section>
-<!-- ─── About Me Section ─── -->    
+
+<!-- ─── Contact Me Section ─── -->    
+
 <section class="contact">
-    
+    <div class="end-text">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+    </div>
+    <form action="https://formspree.io/f/mqkoarkl" method="post" id="contact-form">
+        <input type="text" name="name" id="name" maxlength="30" placeholder="Full Name" required>
+        <input type="email" name="email" id="email" placeholder="Email Address" required>
+        <textarea type="text" name="msg" id="msg" cols="30" rows="5" maxlength="500" placeholder="Write a message here" required></textarea>
+        <button class="action-btn" id="submit-btn" type="submit">Get in touch</button>
+    </form>
+    <div id="shape"><img scr="images/Shape@2x.png"></div>
```

Comment:
- [ ] [OPTIONAL] I see your creativity here but it will be better to style this with an empty `div` by declaring a `width`, `height`, `background-image`, and `border-radius`. This particular element is neither an image content that gives additional meaning to the page nor is it a background. I think not using an image tag will a really cool way to save space, keep code less littered and show your creative strength. How is that? 😄